cibuild.cmd

```diff
@@ -55,6 +53,7 @@ git show --no-patch --pretty=raw HEAD
 
 REM Restore the NuGet packages
 call "%RoslynRoot%\Restore.cmd" || goto :BuildFailed
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :BuildFailed
```

Comment:
Ideally we would not needs this call here, since we call it in `Restore.cmd`. However, we seem to lose the environment state (probably because of the call to `exit /b 0`) so we need to call it again here.